django/thunderstore/account/api/experimental/views.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.authtoken.models import Token
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class TokenExperimentalApiView(APIView):
```

Comment:
DRF also has a `CreateAPIView` which could be used, but in this case it might not make sense and just plain `APIView` might even work better.

Reply:
I think `APIView` works better here